eni/error.go

```diff
@@ -0,0 +1,11 @@
+package eni
+
+import "github.com/giantswarm/microerror"
+
+var invalidConfigError = microerror.New("invalid config error")
+
+var executionFailedError = microerror.New("execution failed error")
+
+var eniNotAttachedError = microerror.New("eni not attached error")
+
+var eniNotDetachedError = microerror.New("eni not detached error")
```

Comment:
Those two errors should be dropped and replaced with `executionFailedError`. Especially when they don't have matchers.